assets/js/common-components/b4/lineItems.tpl.liquid

```diff
@@ -0,0 +1,114 @@
+<div class="card-body py-2 border-gray-200 text-condensed font-weight-bold">
+    <div class="row align-items-center">
+        <div class="col">{{ 'customer.order.product' | t }}</div>
+        <div class="col-5">
+            <div class="row align-items-center">
+                <div class="col text-center">{{ 'customer.order.quantity' | t }}</div>
+                <div class="col text-right">{{ 'customer.order.total' | t }}</div>
+                <div class="col-5" ng-if="!$ctrl.hideReorder"><span></span></div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div ng-repeat="group in $ctrl.order.configuredGroups" class="card-body py-2 border-gray-200 border-top border-1">
```

Comment:
I don't think every of group and line item is card-body, instead here should be only single card body and then rows and columns or, I think it's better, we may use list groups here: https://getbootstrap.com/docs/4.6/components/card/#list-groups

Reply:
@asvishnyakov Then we'll have to remove paddings from that single 'card-body' sides and then apply side paddings to every group <div> in order to keep full-width bottom border on each group. I don't think this class is so semantic (adds only paddings), so i'd leave it as is. As example, we can have an order with just one configured group and nothing else, so this card body will be the only one inside ;)